src/utils/escape.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+declare var CSS: Object
+
+const escapeRegex = /[[\].#*$><+~=|^:(),"'`-]+/g
+const dashesAtEnds = /(^-|-$)/g
+
+const CSS_ESCAPE_AVAILABLE = typeof CSS !== 'undefined' && 'escape' in CSS
```

Comment:
I’m a bit anxious about adding this now without tests :/ can you add some or remove this for now please?

I’m sure we’ll need it more in the future but we’ll need to polyfill it; so right now this switch could lead to some unexpected consequences

Reply:
@philpl Would you be ok with me testing by loading this polyfill? https://github.com/mathiasbynens/CSS.escape

Reply:
Or I can just drop that commit. It'll be a while until browser support is good enough that we'd be able to drop the regex anyways

Reply:
Let’s drop it for now :) but please leave a comment with a note there, saying that it’s a good place to try replacing that piece of code with the polyfill in the future